q2_diversity_lib/citations.bib

```diff
@@ -7,5 +7,37 @@ @article{faith1992conservation
   pages={1--10},
   year={1992},
   publisher={Elsevier},
-  doi={https://doi.org/10.1016/0006-3207(92)91201-3}
+  doi={10.1016/0006-3207(92)91201-3}
+}
+
+@article{pielou1966measurement,
+  title={The measurement of diversity in different types of biological collections},
+  author={Pielou, Evelyn C},
+  journal={Journal of theoretical biology},
+  volume={13},
+  pages={131--144},
+  year={1966},
+  publisher={Elsevier},
+  doi={10.1016/0022-5193(66)90013-0}
+}
+
+% TODO: Selected during a preliminary assay of the literature - must be confirmed/augmented
```

Comment:
I think this one was good.

Reply:
@ebolyen , I'm keeping Shannon. If you're on board, I would like to keep this TODO as well, as a reminder to augment Shannon with a citation that applies entropy to computational biology. That research won't be possible for 19.10, but I think it's still worth doing.

Reply:
sounds good!